vector/src/main/java/im/vector/home/BadgeUpdateListener.kt

```diff
@@ -0,0 +1,9 @@
+package im.vector.home
+
+interface BadgeUpdateListener {
+    fun onBadgeUpdate(count: Int)
+}
+
+interface CommunicateTabBadgeUpdateListener {
```

Comment:
Should this be in a separate file called CommunicateTabBadgeUpdateListener.kt ?

Reply:
I could put that in a separate file. I just thought the purpose of these two listeners is exactly the same. One is used for the custom view to fragment and another one is from fragment to parent fragment. In Kotlin, the filename can be different and the same file can have multiple public (default) class/interface.